### PR TITLE
Deprecate validate-with unset, set to unknown schema, or set to "none"

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by nrwahl2** [comment]
Needs rebase on main

**Comment by kgaillot** [comment]
rebased on current main

File: lib/common/schemas.c [hunk 1]
```diff
@@ -777,14 +777,17 @@ pcmk__validate_xml(xmlNode *xml_blob, const char *validation,
...
 
     entry = pcmk__get_schema(validation);
-    if (entry != NULL) {
-        schema = entry->data;
-        return validate_with(xml_blob, schema, error_handler,
-                             error_handler_context);
+    if (entry == NULL) {
+        pcmk__config_err("Cannot validate CIB with " PCMK_XA_VALIDATE_WITH
+                         " set to an unknown schema such as '%s' (manually "
+                         " edit to use a known schema)",
```
Comment (clumens) [inline lib/common/schemas.c:786]: Extra space between `manually` and `edit`.

Comment (kgaillot) [reply lib/common/schemas.c:786]: thx, fixed

**Review by clumens — APPROVED** [review]

**Pull request merged**